packages/@ionic/cli/src/lib/integrations/capacitor/index.ts

```diff
@@ -190,7 +191,16 @@ export class Integration extends BaseIntegration<ProjectIntegration> {
 
     debug('Getting config with Capacitor CLI: %O', args);
 
-    const output = await this.e.shell.cmdinfo('capacitor', args, { cwd: this.root });
+    const output = await ((cmd: string, opts: ExecOptions): Promise<string | undefined> => {
+      return new Promise((resolve, reject) => {
+        exec(cmd, opts, (error, stdout, stderr) => {
+          if (error) {
+            reject();
```

Comment:
Optional: Do we need an error message here? Or is that done elsewhere?

Reply:
It will actually bubble up to [line 134](https://github.com/ionic-team/ionic-cli/blob/ffd1960d28b0233f97b91b002d00c3ba1801bb8e/packages/%40ionic/cli/src/index.ts#L134) of the main executing script where it will use the default error if it doesn't match specifics, so in this case the default would be what wee want as it would be something akin to `no command 'capacitor' found`